packages/crypto/src/cbor-js.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "cbor-js" {
+  export function encode(value: any): ArrayBuffer;
+  export function decode(data: ArrayBuffer): any;
+}
```

Comment:
I don't really get why this is here. Isn't this something we should import instead of vendoring/checking in?

Here they are: https://www.npmjs.com/package/@types/cbor-js